src/main/webapp/app/core/user/settings/passkey-settings/util/credential-option.util.ts

```diff
@@ -20,9 +20,5 @@ export function createCredentialOptions(options: PublicKeyCredentialCreationOpti
             ...credential,
             id: decodeBase64url(credential.id),
         })),
-        authenticatorSelection: {
```

Comment:
we did overwrite the server `authenticationSelection` here, did not fail as this were the proper values right now, but would fail if we changed it to a more strict version (`required`) - now relying on spread operator, see above lines of code